fitapp/models.py

```diff
@@ -13,6 +13,9 @@ class UserFitbit(models.Model):
     access_token = models.TextField()
     auth_secret = models.TextField()
     refresh_token = models.TextField()
+    # We will store the timestamp float number as it comes from Fitbit here
+    expires_at = models.FloatField()
+    timezone = models.CharField(max_length=128)
```

Comment:
@brad I see that the migration file for the `timezone` field has `default='UTC'`. Did you remove `default='UTC'` here after the migration file was created? If not, how `default='UTC'` was added to the migration file?